docs/Token_Assumptions.md

```diff
@@ -0,0 +1,21 @@
+_Attestation Token use cases & assumptions_
+The following captures what is assumed about the nature of the attestation tokens received for verification.
+
+A token is presented as a single 'blob' (byte stream).
+
+Tokens are presented individually. If a batch mode submission is required, that is the responsibility of an API tier.
+
+Encoding in transit is the responsibility of the API tier, the verification pipeline works on a byte stream.
```

Comment:
It'd be nice if we say what kind of encoding are we talking about here?  My guess is any kind of lossless compression (e.g., what is described by HTTP's `Content-Encoding` header) but you may have another thing in mind?

Reply:
I deliberately generalised because the API call could be in all kinds of context e.g. local API call rather than via REST